<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="editRedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="editGreentLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="editBlueLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renderShapeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="RenderWidget" name="renderWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>971</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="editRed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="editGreen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="editBlue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="editAlpha">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="editAlphaLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importObjFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Obj File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateLeft">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateRight">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateTop">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateDown">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Down</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">renderwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
